pkg/decode/decode_protobuf.go

```diff
@@ -141,9 +141,12 @@ func RecordToMap(fr *model.Record) config.GenericMap {
 			out["XlatSrcAddr"] = model.IP(fr.Metrics.AdditionalMetrics.TranslatedFlow.Saddr).String()
 			out["XlatDstAddr"] = model.IP(fr.Metrics.AdditionalMetrics.TranslatedFlow.Daddr).String()
 		}
-		if fr.Metrics.AdditionalMetrics.FlowEncrypted || fr.Metrics.AdditionalMetrics.FlowEncryptedRet != 0 {
-			out["IPSecSuccess"] = fr.Metrics.AdditionalMetrics.FlowEncrypted
-			out["IPSecRetCode"] = fr.Metrics.AdditionalMetrics.FlowEncryptedRet
+		if fr.Metrics.AdditionalMetrics.IpsecEncryptedRet != 0 {
+			out["IPSecRetCode"] = fr.Metrics.AdditionalMetrics.IpsecEncryptedRet
+			out["IPSecStatus"] = "error"
+		} else if fr.Metrics.AdditionalMetrics.IpsecEncrypted {
+			out["IPSecRetCode"] = int32(0)
+			out["IPSecStatus"] = "success"
```

Comment:
I see you don't return Boolean anymore ?

Reply:
you should just run `make update-config` from the cli repo once the operator PR is merged and it should be good :ok_hand: